content/api/cypress-api/require.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Cypress.require
+---
+
+`Cypress.require` is used with [`cy.origin()`](/api/commands/origin) to utilize
```

Comment:
Seems kind of like using a word in its own definition, since the method is `Cypress.require`. Hopefully it's self-evident that it _requires_ dependencies.

Reply:
Yeah, i didn't like the 'used' and then 'utilize' verbiage, but I'm fine with it.

Reply:
Yeah, that's fair. I caught myself multiple times trying to write `utilize A to utilize B` while writing these docs. Maybe it just needs a completely different sentence structure...

Reply:
I reworked it in https://github.com/cypress-io/cypress-documentation/pull/4681/commits/f38e9b5594c74ca9ccd1658b92a8195675fad5f0